dangerzone/container_utils.py

```diff
@@ -120,30 +127,46 @@ def get_expected_tag() -> str:
 
 def load_image_tarball() -> None:
     log.info("Installing Dangerzone container image...")
-    p = subprocess.Popen(
-        [get_runtime(), "load"],
-        stdin=subprocess.PIPE,
-        startupinfo=get_subprocess_startupinfo(),
-    )
-
-    chunk_size = 4 << 20
-    compressed_container_path = get_resource_path("container.tar.gz")
-    with gzip.open(compressed_container_path) as f:
-        while True:
-            chunk = f.read(chunk_size)
-            if len(chunk) > 0:
-                if p.stdin:
-                    p.stdin.write(chunk)
+    tarball_path = get_resource_path("container.tar")
+    with open(tarball_path) as f:
+        try:
+            res = subprocess.run(
```

Comment:
Would need to check on windows and macOS, but there a possibility that you don't even need to untar the file.

In this case, just passing its location to the `{podman,docker} load -i` invocation would be enough (so no need to `open` the tarball path

Reply:
Oh yeah, you're right. I'll just pass the file location with `-i <path>`.

Reply:
Done in c0ff351a2ed5708b6dd548a8c551f594ec0ee9e7